src/site/stages/build/process-cms-exports/generator-cms-content-model/templates/transformed-schema

```diff
@@ -1,15 +1,18 @@
 module.exports = {
   type: 'object',
   properties: {
+    contentModelType: { enum: ['<%= contentModelType %>'] },
     entity: {
       type: 'object',
       properties: {
+        entityType: { enum: ['<%= baseType %>'] },
+        entityBundle: { enum: ['<%= subType %>'] },
```

Comment:
I don't think all transformers will include these, but they don't seem to be part of required so it should be ok.

Reply:
Yeah, even if they're not used, I figure it won't do any harm to throw them in anyhow. Worst case scenario I can think of, it uses like 1 Kb more in RAM during the build. :man_shrugging: 

Also, we can remove them if we don't like them.